iiasa/message_doc
# Add new logo and diagram to Sphinx/RTD docs

This PR adds the new `message_doc` logo to the documentation and is the corresponding PR to https://github.com/iiasa/message_data/pull/368 and https://github.com/iiasa/message_ix/pull/597.

## How to review

- Read the diff and note that the CI checks all pass.
- Run a specific code snippet or command and check the output.
- View the RTD docs preview at ... look at:
  - the combined IIASA-`message_doc` logo top left,
  - the favicon, and

## PR checklist
